businfo/models/model.go

```diff
@@ -0,0 +1,46 @@
+package models
+
+type BusInfo struct {
```

Comment:
Since model is associated with data stored in the DB, can you research a term, "dto"? I don't think we'll need to store all this data into the DB.